packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -49,12 +51,20 @@ const ExperimentMainContent = ( {
 		return <Spinner />;
 	}
 
+	const label = navigationMenu
+		? sprintf(
+				/* translators: %s: The name of a menu. */
+				__( 'Navigation menu: %s' ),
+				navigationMenu.title
+		  )
+		: __( 'You have not yet created any menus. Displaying a list of your Pages' );
```

Comment:
The piece that's missing is that this doesn't tell the user what kind of UI they're going to encounter. It might be worth keeping something closer to 'Block navigation structure' or 'List view' as the label in that respect. Possibly 'Navigation block list view'.

My feeling is that the context about the current state of the menu could be added as an aria description, as  this is secondary information.

Reply:
Yes. I do wonder whether mirroring what you did in https://github.com/WordPress/gutenberg/pull/47047 would be a good option. Use an description to announce what the thing does.

This definitely requires accessibility feedback from an expert in this field.